### PR TITLE
Add release notes for 0.195

### PR DESCRIPTION
**Pull request by raghavsethi**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.195.rst [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+=============
+
+General Changes
+---------------
+
+* Fix :func:`histogram` for map type when type coercion is required.
+* Fix ``nullif`` for map type when type coercion is required
+* Fix incorrect termination of queries when the coordinator to worker communication is under high load.
+* Fix race condition that causes queries with a right/full outer join to fail.
+* Change reference counting for complex varchar, varbinary, and complex types to be approximate. This
```
Comment (wenleix) [inline presto-docs/src/main/sphinx/release/release-0.195.rst]: I assume the first "complex" is not necessary? There is no "complex varchar". 

```
... for varchar, varbinary, and complex types to be approximate.
```

File: presto-docs/src/main/sphinx/release/release-0.195.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.195
+=============
+
+General Changes
+---------------
+
+* Fix :func:`histogram` for map type when type coercion is required.
```
Comment (wenleix) [inline presto-docs/src/main/sphinx/release/release-0.195.rst]: What's this?

File: presto-docs/src/main/sphinx/release/release-0.195.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+* Fix ``nullif`` for map type when type coercion is required
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.195.rst]: End in period

File: presto-docs/src/main/sphinx/release/release-0.195.rst [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+* Fix incorrect termination of queries when the coordinator to worker communication is under high load.
+* Fix race condition that causes queries with a right/full outer join to fail.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.195.rst]: "right or full"

File: presto-docs/src/main/sphinx/release/release-0.195.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+-------------
+
+* Fix an error that can occur while listing tables if one of the listed tables is dropped.
+
+Hive Changes
+------------
+
+* Add support for LZ4 compressed ORC files.
+* Add support for reading Zstandard compressed ORC files.
+* Valid ORC compression block size when reading ORC files.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.195.rst]: "Validate"

**Pull request closed without merging**